rs/sns/swap/proto/ic_sns_swap/pb/v1/swap.proto

```diff
@@ -200,8 +200,8 @@ message Swap {
   // Specified on creation. That is, always specified and immutable.
   Init init = 1;
 
-  // Specified in the transition from PENDING to OPEN and immutable
-  // thereafter.
+  // Derived from `init`, always specified and immutable. In most cases `init`
+  // should be used instead.
```

Comment:
I thought you said that this is redundant vs. init? If so, shouldn't this instead say that this field is deprecated, and recommend users use init instead?

Reply:
True, it should, but I think that instead of saying it's deprecated we should actually deprecate it by saying `[deprecated = true]`. And then we would get a lot of annoying errors from the places where the code uses `init` when it should use `params`. So that was why I didn't actually deprecate it. I think we should do that at some point soon but I didn't want to do it in this MR

Reply:
I can go either way.

Adding a comment now seems pretty easy.

Adding `[deprecated = true]` is probably a little disruptive, so I understand why not add it now.

I doubt we'll ever get to that ticket. We should avoid creating such tickets. I believe avoiding is what Georgi, Lara, and Max want.

Reply:
I expect we will include that ticket as part of the "Swap canister upgrades" roadmap item